proto/sanitizer.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+package pluginextensionv2;
+
+option go_package = ".;pluginextensionv2";
+
+message SanitizeRequest {
+  string filename = 1;
+  bytes content = 2;
+  bytes domPurifyConfig = 3;
+  bool allowAllLinksInSvgUseTags = 4;
```

Comment:
maybe we have this proto definition be more "open" to non svg thigns in the future?

Perhaps 
```suggestion
  string sanitizer = 3; // 'svg' | ...
  bytes sanitizerConfig = 4; // json blob that depends on the sanitier type
```

Reply:
yeah - thats better. For `string sanitizer` I would probably use the library name or "sanitization logic type" instead of the file type so we avoid having an abstraction over the library config. DOMPurify has a pretty extensive API and it can support html, svg or mathml out of the box. Soo, something like this:


```
sanitizer: 'dompurify'
config: { dompurifyConfig: { ... }, domPurifyExtraFunctionalitiesBuiltByUsConfig: { ... } }
```
and

```
sanitizer: 'some-other-image-sanitizer'
config: { some-other-image-sanitizerConfig': { ... }}
```

what do you think?